### PR TITLE
Scroll to top button added

### LINKED ISSUE #212: Add scroll to top Button
When we are at the bottom of the page, it is difficult for a user to add up and up. Therefore we need a scroll to top floating button.

- **Place:** Bottom Right
- **Button Color:** #1d1283
- **Icon Color:** #fff

**Note:** Make it responsive for all the devices

**Image for Reference**
image

### PR DESCRIPTION
**Pull request by anmoldewanrai**
### What is the change?

Added a scroll to top button 

### Is Development Tested?

- [x] Yes
- [ ] No

### Before / After Change Videos

Before :

https://user-images.githubusercontent.com/52621609/149350081-04c31c0f-d33a-4870-8702-e46afe2b4c60.mp4

After :

https://user-images.githubusercontent.com/52621609/149350992-f748d564-9670-48fb-ac8c-b8d4f6217137.mp4

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
   "editor.formatOnSave": true,
-  "editor.defaultFormatter": "esbenp.prettier-vscode"
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "liveServer.settings.port": 5502
```
Comment (rohan09-raj) [inline .vscode/settings.json:4]: Can we please remove this ?? as default port is 5500 for live server

File: index.html [hunk 2]
```diff
@@ -58,6 +58,23 @@
...
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M2 9L9 2L16 9"
+        stroke="white"
+        stroke-width="4"
+        stroke-linecap="round"
+        stroke-linejoin="round"
+      />
```
Comment (rohan09-raj) [inline index.html:75]: Can we please move this to image folder and use a Image tag here ??

**Review by rohan09-raj — COMMENTED** [review]
Thanks for the PR, can you please check why tests are failing.
Do you forget to run something which is mentioned in `README.md` ??
Also please do check why tests failed in the details section
image

File: index.html (hunk 2 above)
Comment (anmoldewanrai) [reply index.html]: Image moved to the **img** folder and used an **** tag instead

**Review by rohan09-raj — APPROVED** [review]
Looks good to me 🥳, Good work @anmoldewanrai

**Pull request merged**